pants.toml

```diff
@@ -91,10 +97,13 @@ experimental_lockfile = "src/python/pants/backend/python/lint/docformatter/lockf
 
 [flake8]
 config = "build-support/flake8/.flake8"
+# TODO: Write a script to generate all our tool lockfiles. We need to disable custom requirments like this and also
+#  set the ICs to something useful for our users.
```

Comment:
This is my next PR :)

Reply:
Welp, that sucks. I had it all ready, but didn't push yet to GitHub. My laptop broke this morning and Apple had to purge the machine, so gotta restart now :)